CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+# Contributing to Bandit
+Thanks for considering take part in improvement of Bandit project, contributions are always welcome!  
+Here are guidelines and rules that can be helpful if you plan to want to get involved in the project.
+
+#### Table Of Contents
+[Code of Conduct](#code-of-conduct)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+  * [Things You Should Know Before Getting Started](#things-you-should-know-before-getting-started)
+
+## Code of Conduct
+Everyone who participates in this project is governed by [Bandit Code of Conduct](https://github.com/Glyphack/bandit/blob/master/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct).
+## Reporting Bugs
+If you encountered a bug and want to let us know use [GitHub issues](https://guides.github.com/features/issues/).
+
+**Before submitting a new issue** you might want to check [existsing issue](https://github.com/PyCQA/bandit/issues) to know if there is alredy reported issue and the issue is still open
+in that case you can add a comment to the issue instead of creating a new one.
```

Comment:
in that case you can add a comment to the **existing** issue instead of creating a new one.